go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/rjkroege/edwood
 go 1.11
 
 require (
-	9fans.net/go v0.0.0-20181112161441-237454027057
+	9fans.net/go v0.0.2
```

Comment:
can you revert this or make it a different CL

Reply:
Done.